assets/app/scripts/app.js

```diff
@@ -286,7 +286,7 @@ angular
       });
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
-  .constant("APIS_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).apis))
+  .constant("APIS_CFG", (window.OPENSHIFT_CONFIG || {}).apis)
```

Comment:
was there a good reason to clone these anyway or should I just make all of them reference the actual config object